docs/src/pages/demos/menus/MenuListItemStyle.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MenuList } from 'material-ui/Menu';
+import Paper from 'material-ui/Paper';
+import { withStyles } from 'material-ui/styles';
+import { ListItem, ListItemText } from 'material-ui/List';
+
+const styles = {
+  button: {
+    '&:focus, &:focus>div>h3': {
+      backgroundColor: '#2196f3',
+      color: 'white',
+    },
+  },
+};
+
+const MenuListItemStyle = ({ classes }) => (
+  <Paper>
+    <MenuList>
+      {['First', 'Second', 'Third'].map(item => (
```

Comment:
Please could you use real-world example labels? (You can repeat those in the other examples, or refer to the spec for inspiration.)